modules/icgc-argo-workflows/prep/metrics/resources/usr/bin/rnaaln.py

```diff
@@ -29,17 +29,15 @@
 tool_fieldmap = { # 'target name' : 'original name'
     'hisat2': {
         'paired_total' : 'paired_total',
-        'unpaired_total' : 'unpaired_total',
-        'overall_alignment_rate' : 'overall_alignment_rate'
+        'unpaired_total' : 'unpaired_total'
     },
     'star' : {
         'total_reads' : 'total_reads',
         'avg_input_read_length' : 'avg_input_read_length',
-        'uniquely_mapped_percent' : 'uniquely_mapped_percent',
+        'pct_uniquely_mapped' : 'uniquely_mapped_percent',
```

Comment:
We handle `percent` as `pct` for standardization. This one was missed. Now correct it to the standard format